### PR TITLE
Add deprecation message for slss alias

### PR DESCRIPTION
**Pull request by crash7**
Addresses: #8122

### CODE REVIEW

File: scripts/serverless.js [hunk 1]
```diff
@@ -36,6 +36,13 @@ if (nodeVersion < 10) {
   );
 }
 
+if (/slss$/i.test(process.argv[1])) {
+  require('../lib/utils/logDeprecation')(
+    'DEPRECATED_ALIAS',
+    'Please use sls or serverless commands. Support for slss command will be removed with v2.0.0'
```
Comment (medikoo) [inline scripts/serverless.js:42]: Let's maybe reword it to:

```
Support for "slss" command will be removed with v2.0.0. Use "sls" or "serverless" instead
```

File: scripts/serverless.js [hunk 2]
```diff
@@ -36,6 +36,13 @@ if (nodeVersion < 10) {
   );
 }
 
+if (/slss$/i.test(process.argv[1])) {
```
Comment (medikoo) [inline scripts/serverless.js:39]: Let's add it here: https://github.com/serverless/serverless/blob/c6e25bfc2008ec492a212574972feb6bfcf2f94b/bin/serverless.js#L12

**Review by medikoo — CHANGES_REQUESTED** [review]
Thank you! That looks promising. Please see my remarks. 

We need to also document this deprecation here: https://github.com/serverless/serverless/blob/master/docs/deprecations.md

**Comment by medikoo** [comment]
I've just realized that this detection won't work on windows (`process.argv[1]` will always point resolved script filename).

Having that I believe we need to introduce a temporary `slss.js` file in `bin` folder, put deprecation there, make it require `serverless.js` and point `slss` to it.

**Comment by crash7** [comment]
> I've just realized that this detection won't work on windows (`process.argv[1]` will always point resolved script filename).
> 
> Having that I believe we need to introduce a temporary `slss.js` file in `bin` folder, put deprecation there, make it require `serverless.js` and point `slss` to it.

Good catch, totally forgot about windows. Going to add the dummy script then.

File: bin/slss.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env node
+
+// TODO (BREAKING): Remove this file with next major release
+
+'use strict';
+
+require('../lib/utils/logDeprecation')(
+  'DEPRECATED_ALIAS_SLSS',
```
Comment (medikoo) [inline bin/slss.js:8]: Let's not use `DEPRECATED_` prefixes in deprecation codes. They seems superfluous. I guess just `SLSS_CLI_ALIAS` would make a great name

File: docs/deprecations.md [hunk 4]
```diff
@@ -39,3 +39,9 @@ Please use `onUnauthenticatedRequest` instead. `allowUnauthenticated` will be re
 ## `bin/serverless`
 
 Please use `bin/serverless.js` instead. `bin/serverless` will be removed with v2.0.0
+
+<a name="DEPRECATED_ALIAS_SLSS"><div>&nbsp;</div></a>
```
Comment (medikoo) [inline docs/deprecations.md:43]: Let's add new deprecation as _first_ (see https://github.com/serverless/serverless/blob/master/CONTRIBUTING.md#configuring-deprecations).

Ideally newest are at the top, oldest (most likely least relevant) at the bottom

**Review by medikoo — CHANGES_REQUESTED** [review]
@crash7 thanks that looks good, I have just two improvement proposals. Please see my comments

File: docs/deprecations.md (hunk 4 above)
Comment (crash7) [reply docs/deprecations.md]: Sorry, I didn't know (and now that you said it, it makes a lot of sense).

**Review by medikoo — APPROVED** [review]
Looks great! Thank you @crash7
